src/main/java/games/strategy/triplea/delegate/BattleTracker.java

```diff
@@ -32,6 +32,8 @@
 import games.strategy.sound.SoundPath;
 import games.strategy.triplea.Constants;
 import games.strategy.triplea.TripleAUnit;
+import games.strategy.triplea.ai.proAI.ProData;
+import games.strategy.triplea.ai.proAI.util.ProBattleUtils;
```

Comment:
We really shouldn't be using AI classes outside of the AI as it adds a lot of coupling and increases risk when updating the AI. If we want to use code from the AI outside of it then we should look to extract it into utility classes.

@DanVanAtta Do you agree with that? I don't think we should have merged this without addressing that.